src/CycleWithDestructor.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Brick\WeakmapPolyfill;
```

Comment:
This repo does not belong to Brick, let's just use:

```suggestion
namespace WeakmapPolyfill;
```

Reply:
adressed